napari/_qt/utils.py

```diff
@@ -18,6 +18,20 @@
 
 from ..utils.misc import is_sequence
 
+QBYTE_FLAG = "!QBYTE_"
+
+
+def is_qbyte(string):
+    return isinstance(string, str) and string.startswith(QBYTE_FLAG)
+
+
+def qbytearray_to_str(qbyte):
+    return QBYTE_FLAG + qbyte.toBase64().data().decode()
+
+
+def str_to_qbytearray(string):
+    return QByteArray.fromBase64(string.lstrip(QBYTE_FLAG).encode())
```

Comment:
Incorrect, you may lstrip too much.
```
In [1]: QBYTE_FLAG = "!QBYTE_"

In [2]: "BYE_QT".lstrip(QBYTE_FLAG)
Out[2]: ''
```

Reply:
This makes the assumption that qbytearra_to_str was used so that string is supposed to be

`"!QBYTE_somebase64datahere"`

Reply:
yeah but "BYEBYEQT" is a valid base64 string.... so QBYTE

```
In [7]: qb.toBase64().data().decode()
Out[7]: 'BYEBYEQT'

In [8]: type(qb)
Out[8]: PyQt5.QtCore.QByteArray
```